### DISCUSSION TITLE
How to ship multiple apps from same codebase ?

### DISCUSSION THREAD

**Discussion by sharvilak11** [expo/expo#20295] [Archived: Questions]
We are building an app which needs to be customised from assets and global variables for each or our client. What's the best way to publish different apps from the same code ? 

I tried to use `app.config.js` to dynamically produce the config with different package name and other values, but the app is ejected and when running the build it says `Specified value for "android.package" in app.config.js or app.json is ignored because an android directory was detected in the project. EAS Build will use the value found in the native code.`

Is there a way to achieve this ?

**Comment by roo12312** [comment]
any answer?

**Comment by gustavo-bonfim** [comment]
I don't know the best way to do this, but perhaps you could manipulate app.config.js and use `npx expo prebuild --clean` before each build to redo the native code folders.

After that I believe EAS build will work fine. Worth trying :)
